src/microbe_stage/MicrobeStage.cs

```diff
@@ -269,7 +267,6 @@ public void OnReturnFromEditor()
 
         HUD.OnEnterStageTransition();
         HUD.HideReproductionDialog();
-        HUD.UpdatePatchInfo(GameWorld.Map.CurrentPatch.Name);
```

Comment:
I'm pretty sure removing this causes issues.

Reply:
There is already another one on this line, which is why it's a duplicate, and so I removed it.
https://github.com/Revolutionary-Games/Thrive/blob/4a2d03c2551530f5656a5a76a3d4ca406af22206/src/microbe_stage/MicrobeStage.cs#L244

Reply:
I see, I did not load the previous code for the diff to look for that.